### PR TITLE
Add sharing buttons for Email and Whatsapp

### PR DESCRIPTION
**Pull request by pixeltrix**
Adds the additional sharing buttons for Email and Whatsapp. Icon width was set to 19px so that the labels line up.

https://www.pivotaltracker.com/story/show/95724426

### CODE REVIEW

**Comment by otlaitil** [comment]
Looks good!

**Comment by davidbasalla** [comment]
+1

File: app/views/shared/_share_petition.html.erb [hunk 1]
```diff
@@ -8,9 +8,21 @@
     <% end %>
   </li>
   <li>
+    <%= link_to "mailto:?subject=#{URI.escape(title)}&body=#{URI.escape(petition_url(petition))}" do %>
+      <span class="icon icon-share-email"></span>
```
Comment (alexander-bobin) [inline app/views/shared/_share_petition.html.erb]: I should have mentioned, we need to add `arial-hidden="true"` to our icons.

**Comment by alexander-bobin** [comment]
LGTM other than `aria-hidden`

**Comment by alexander-bobin** [comment]
LGTM :shipit:

**Pull request merged**